### DISCUSSION TITLE
I can see entries but can't create new ones after restoring my backup password-store

### DISCUSSION THREAD

**Discussion by oren** [zeapo/Android-Password-Store#1252] [Q&A] [answered]
after my laptop crashed i setup pass on a new laptop but I am not able to create new entries in pass. i can only read. when i tried to generate a new entry i got: 
```
Error: You must run:
    pass init your-gpg-id
before you may use the password store.
```

gpg -k    
```
/home/oren/.gnupg/pubring.kbx
-----------------------------
pub   rsa4096 2020-02-25 [SC]
      xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
uid           [ unknown] my name <my-email@gmail.com>
sub   rsa4096 2020-02-25 [E]
```

I assume my id is my email so I did:
```
pass init my-email@gmail.com
```
and now i see:
```
gpg: xxxxxxxxxxxxxx: There is no assurance this key belongs to the named user
gpg: [stdin]: encryption failed: Unusable public key
Password encryption aborted.
```

**Comment by msfjarvis** [comment, marked as answer]
Please directly open a discussion for things that are not directly about APS.

You're seeing this because your key is not trusted, and you need to do `gpg --edit-key <key id>`, followed by typing `trust` and `5` to ultimately trust it.
